### PR TITLE
rebase onto implemented

### PR DESCRIPTION
**Pull request by Qusijue**

### CODE REVIEW

**Comment by pcottle** [comment]
Wow exciting you took this on! Would you like to add some tests like in these files:
https://github.com/pcottle/learnGitBranching/blob/main/__tests__/git.spec.js#L170

or should I? If you call `debug_copyTree()` in the JS console while using the app, you can get the JSON that represents the current state of the tree (which is how I wrote all those tests)

File: __tests__/git.spec.js [hunk 1]
```diff
@@ -423,4 +423,27 @@ describe('Git', function() {
...
+      );
+    });
+
+    it('rebase onto fast forward', function() {
+      return expectTreeAsync(
+        'git switch -c F1; git commit; git switch -c F1; git commit;',
```
Comment (pcottle) [inline __tests__/git.spec.js:444]: this doesnt seem to have a `rebase --onto` call?

**Comment by pcottle** [comment]
Thanks for adding tests though! Let me know if you want to update that one or we can just land as-is

**Comment by Qusijue** [comment]
the test has been fixed

**Comment by pcottle** [comment]
👏 👏 👏

**Comment by pcottle** [comment]
(updating `gh-pages` now with your changes!)

**Pull request merged**